docs/storage.md

```diff
@@ -184,4 +185,22 @@ config:
     storage_account: <Name of Azure Storage Account>
     storage_account_key: <Storage Account key>
     container: <Blob container>
-```
\ No newline at end of file
+```
+
+### OpenStack Swift Configuration
+Thanos uses [gophercloud](http://gophercloud.io/) client to upload Prometheus data into [OpenStack Swift](https://docs.openstack.org/swift/latest/).
+
+Below is an example configuration file for thanos to use OpenStack swift container as an object store. 
+
+```yaml
+type: SWIFT
+config:
+    authUrl: <identity endpoint aka auth URL>
+    username: <username>
+    password: <password>
+    tenantName: <tenantName>
+    regionName: <region>
+    containerName: <container>
```

Comment:
can we update this to `snake_case`

Reply:
@domgreen sorry, missed this one in the last commit. fixed it now.